### ISSUE TITLE
WCAG2ICT Project: Modify 2.4.7 Focus Visible (Now level A)

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag2ict#36]
- [ ] Make any needed changes to [2.4.7 Focus Visible](https://www.w3.org/TR/WCAG22/#focus-visible) section (from WCAG 2.2)
- [ ] Definitions: "Focus indicator" should be added (if this SC is done before 2.4.11 Focus Appearance). Though this term as of 21 Feb. 2023 has no web-centric content in the definition.
- [ ] Make any modifications to the understanding from the [Understanding 2.4.7 Focus Visible](https://www.w3.org/WAI/WCAG22/Understanding/focus-visible.html) Intent and Benefits sections

**Comment by maryjom** [comment]
Looks like WCAG 2.2 chose not to make this SC Level A, so this can be removed from our list of things to do. Closing the issue.
